fractal_tasks_core/tasks/projection.py

```diff
@@ -55,122 +65,65 @@ def projection(
             `create_cellvoyager_ome_zarr_init`.
     """
     method = DaskProjectionMethod(init_args.method)
-    logger.info(f"{init_args.origin_url=}")
-    logger.info(f"{zarr_url=}")
-    logger.info(f"{method=}")
+    ngio_logger.info(f"{init_args.origin_url=}")
```

Comment:
Is the plan to generally transition to using ngio logger in any task that uses ngio?